<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIstemView</class>
 <widget class="QWidget" name="SIstemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>65</horstretch>
    <verstretch>60</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>650</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>195</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>70</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>90</red>
        <green>93</green>
        <blue>161</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>195</red>
        <green>197</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>195</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>70</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>90</red>
        <green>93</green>
        <blue>161</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>195</red>
        <green>197</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>70</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>193</red>
        <green>195</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>70</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>90</red>
        <green>93</green>
        <blue>161</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>70</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>70</green>
        <blue>121</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>135</red>
        <green>140</green>
        <blue>242</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>SIstemView</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyCameraWindow" name="widgetOpenCV" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>510</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>680</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetButton" native="true">
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>680</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QPushButton" name="buttonStart">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string extracomment="Memulai Kendali Tangan">St&amp;art </string>
       </property>
       <property name="icon">
        <iconset resource="projectresource.qrc">
         <normaloff>:/image/images/play.png</normaloff>:/image/images/play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonHistogram">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Histogram</string>
       </property>
       <property name="icon">
        <iconset resource="projectresource.qrc">
         <normaloff>:/image/images/histogram.jpg</normaloff>:/image/images/histogram.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonContour">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>10</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Contour</string>
       </property>
       <property name="icon">
        <iconset resource="projectresource.qrc">
         <normaloff>:/image/images/7023332-human-hand-concept-made-of-100-icons-set-in-black-color.jpg</normaloff>:/image/images/7023332-human-hand-concept-made-of-100-icons-set-in-black-color.jpg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonExit">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>10</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Keluar</string>
       </property>
       <property name="icon">
        <iconset resource="projectresource.qrc">
         <normaloff>:/image/images/exit.png</normaloff>:/image/images/exit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>29</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonStop">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>91</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Stop</string>
       </property>
       <property name="icon">
        <iconset resource="projectresource.qrc">
         <normaloff>:/image/images/stop.png</normaloff>:/image/images/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonPetunjuk">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>10</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Petunjuk</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyCameraWindow</class>
   <extends>QWidget</extends>
   <header location="global">MyCameraWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="projectresource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_buttonStart_clicked()</slot>
 </slots>
</ui>
